src/Utils/util_items.h

```diff
@@ -1,35 +1,259 @@
-#pragma once
-
-#include <string>
-#include "../SDK/definitions.h" //ItemDefIndex
+#include "util_items.h"
 
+#include "util.h"
+#include "../interfaces.h"
 namespace Util
 {
 	namespace Items
 	{
-		/*
-		 * Look up a ItemDefinitionIndex ordinal value by the symbolic name
-		 * using an internal map
-		 */
-		enum ItemDefinitionIndex GetItemIndex(const std::string);
-
-		/*
-		 * Look up the symbolic name of a ItemDefinitionIndex ordinal value
-		 * using an internal map
-		 */
-		std::string GetItemName(const enum ItemDefinitionIndex);
-
-		std::string GetItemEntityName(ItemDefinitionIndex index);
-
-		std::string GetItemConfigEntityName(ItemDefinitionIndex index);
-
-		std::string GetItemDisplayName(ItemDefinitionIndex index);
-
-		bool IsKnife(ItemDefinitionIndex index);
-		bool IsUtility(ItemDefinitionIndex index);
-		bool IsGlove(ItemDefinitionIndex index);
-		bool IsCTWeapon(ItemDefinitionIndex index);
-		bool IsTWeapon(ItemDefinitionIndex index);
-		bool IsScopeable(ItemDefinitionIndex index);
+		using i = ItemDefinitionIndex;
+		const std::map<std::string, ItemDefinitionIndex> ItemNames = {
```

Comment:
Please move all of this back to  src/Utils/util_items.cpp

Reply:
This _might have been_ (deffo mine) my mistake while uploading the code I might have posted in .h instead of .cpp